src/test/java/org/assertj/core/error/ShouldHaveSize_create_Test.java

```diff
@@ -48,4 +52,19 @@ public void should_create_error_message_with_hexadecimal_representation() {
     String message = factory.create(new TextDescription("Test"), new HexadecimalRepresentation());
     assertThat(message).isEqualTo(String.format("[Test] %nExpected size:<2> but was:<4> in:%n<['0x0061', '0x0062']>"));
   }
+
+  @Test
+  public void should_create_error_message_for_file_not_has_size() {
+    // GIVEN
+    ErrorMessageFactory factory = shouldHaveSize(new FakeFile("abc"), 3L);
+    String expectedErrorMessage = format("[Test] %n"
+                                         + "Expecting file%n"
+                                         + "  <abc>%n"
+                                         + "to have size:%n"
+                                         + "  <3L>%n"
+                                         + "but had:%n"
+                                         + "  <0L>");
+    // THEN
+    assertThat(factory.create(new TextDescription("Test"), new StandardRepresentation())).isEqualTo(expectedErrorMessage);
```

Comment:
extract `factory.create(new TextDescription("Test"), new StandardRepresentation())` to a variable and put it in the `// WHEN` section.